tfhe/src/shortint/engine/wopbs/mod.rs

```diff
@@ -155,16 +156,27 @@ impl ShortintEngine {
             &mut self.encryption_generator,
         );
 
+        let max_noise_level = MaxNoiseLevel::new(divide_ceil(
+            parameters.carry_modulus.0 * parameters.message_modulus.0 - 1,
+            parameters.message_modulus.0 - 1,
+        ));
+
```

Comment:
same remark for the noise

maybe rename this one to indicate it's the noise for the wopbs server key

Reply:
Done